lib/linguist/generated.rb

```diff
@@ -95,6 +96,13 @@ def xcode_file?
       ['.nib', '.xcworkspacedata', '.xcuserstate'].include?(extname)
     end
 
+    # Internal: Is the blob part of Pods/, which contains dependencies not meant for humans in pull requests.
+    #
+    # Returns true or false.
+    def cocoapods?
+      !!name.match(/Pods\//)
```

Comment:
This needs tightening as it'll match `I-am-not-Pods/` or any other combination of `*Pods/` which we probably don't want.

Reply:
No problem! 
P.S just so you're aware - this same method applies to node_modules etc, which means this would also ignore a folder called `MyThing-node_modules`

Reply:
Yup, I saw that. I'd like to only concentrate on cocoapods in this PR to keep things simple and limit the scope of the changes to be relevant to the original purpose of this PR.